include/taskmgr.h

```diff
@@ -58,6 +58,7 @@
 #define NET_STACK_SIZE     8192
 #define DEBUG_STACK_SIZE   8192                 // Needs a lot of stack for output if UpdateClockFromWeb is called from debugger
 #define REMOTE_STACK_SIZE  4096
+#define SCREEN_STACK_SIZE  8192
```

Comment:
It may be an idea to only increase the screen stack size to 8K if AMOLED_S3 is defined. The default 2.5K seems to be enough for the other devices, and we're already low on RAM in some of those (the M5StickC Plus with the SPECTRUM config comes to mind).

Maybe like so?

```suggestion
#if AMOLED_S3
    #define SCREEN_STACK_SIZE 8192
#else 
    #define SCREEN_STACK_SIZE DEFAULT_STACK_SIZE
#endif
```